### ISSUE TITLE
Please add spectron testing

### ISSUE THREAD

**Issue by forki** [fable-compiler/fable-elmish-electron-material-ui-demo#9]
Would be awesome to see spectron testing example. https://www.electronjs.org/spectron

**Comment by cmeeren** [comment]
Agreed. This is unfortunately not something I can prioritize working on myself, but anyone feel free to have a look at what would be involved.

**Comment by mardukbp** [comment]
Spectron was [discontinued in 2022](https://www.electronjs.org/blog/spectron-deprecation-notice). This issue can be closed.
